bin/run_tests_for_modules.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "${BASH_SOURCE%/*}/.."
+
+for module in $(echo "$@" | tr , ' '); do
+    if [[ -x "${module}/bin/test.sh" ]]; then
+        "${module}/bin/test.sh"
+    else
+        echo
```

Comment:
Using a warning here is bad:

* It's easy to miss this warning when we expect the module to have tests
* We will get bogus warnings for modules that should not have tests

Let's add an environment variable or another way to do this reliably.

Reply:
Addressed.